app/src/main/java/de/westnordost/streetcomplete/quests/bike_rental_capacity/AddBikeRentalCapacity.kt

```diff
@@ -0,0 +1,38 @@
+package de.westnordost.streetcomplete.quests.bike_rental_capacity
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+
+class AddBikeRentalCapacity : OsmFilterQuestType<Int>() {
+
+    override val elementFilter = """
+        nodes, ways with
+         amenity = bicycle_rental
+         and access !~ private|no
+         and bicycle_rental ~ docking_station|dropoff_point
+         and (!capacity or capacity older today -6 years)
+    """
+
+    override val changesetComment = "Add bicycle rental capacities"
+    override val wikiLink = "Tag:amenity=bicycle_rental"
+    override val icon = R.drawable.ic_quest_bicycle_rental
```

Comment:
As an icon, I used https://github.com/streetcomplete/StreetComplete/blob/v43.0/res/graphics/quest/bicycle_rental.svg for now. We might need to distinguish between bicycle rental capacity (#2129) and type (#3995) in the future, so maybe the icon still needs some work.

Reply:
Maybe if the key was over the top of the bike (or cleverly formed into a wheel/the frame or something fancy), you could double up bikes like the other capacity quests?

Reply:
When/if the other quest type gets added, this icon could just be two keys on top of each other. The background color is already hint enough that it is about bicycles.

Reply:
Done in [`396dfbb`](https://github.com/streetcomplete/StreetComplete/pull/4008/commits/396dfbb7b6805f28de893eaf1e77960f85e177e2).